cli.js

```diff
@@ -42,6 +42,10 @@ const repo = cli.input[0];
 			process.exit(1);
 		}
 
-		await open(url);
+		if (url) {
```

Comment:
Would be more readable to do an early return.

Reply:
fixed